### PR TITLE
RD-2332 Add the Deployment display name column to the Executions widget.

### PR DESCRIPTION
**Pull request by stepek**
image
https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/822/

### CODE REVIEW

**Comment by Gelio** [comment]
I am overwhelmed with code review and switching the development branches to 6.1.0. I cannot review this PR today, maybe I'll do that tomorrow :slightly_smiling_face: 

Looking at the code changes initially, I believe we are missing system tests to verify that the functionality is working

File: test/cypress/integration/widgets/executions_spec.ts [hunk 1]
```diff
@@ -18,6 +18,24 @@ describe('Executions', () => {
...
+                cy.editWidgetConfiguration('executions', () => {
+                    cy.get('div[name="fieldsToShow"]').click();
+                    cy.get('div[option-value="Deployment"]').click();
+                    cy.get('div.header').click();
+                })
+            );
+
+            it('shows deployment display name', () => {
+                cy.log('Check if display name is provided');
+                cy.get('table.executionsTable tr:first-of-type td:nth-of-type(2)').should(
```
Comment (kubama) [inline test/cypress/integration/widgets/executions_spec.ts:32]: You may consider using `getTable` command here

File: test/cypress/integration/widgets/executions_spec.ts [hunk 2]
```diff
@@ -18,6 +18,24 @@ describe('Executions', () => {
     });
 
     describe('in table mode', () => {
+        describe('change visible columns', () => {
+            before(() =>
+                cy.editWidgetConfiguration('executions', () => {
+                    cy.get('div[name="fieldsToShow"]').click();
+                    cy.get('div[option-value="Deployment"]').click();
+                    cy.get('div.header').click();
+                })
```
Comment (kubama) [inline test/cypress/integration/widgets/executions_spec.ts:27]: You may consider setting `fieldsToShow` property right away in `usePageMock` call. The extra column would be then available in all test cases, but I guess there is no issue with that.

File: test/cypress/integration/widgets/executions_spec.ts [hunk 3]
```diff
@@ -18,6 +18,24 @@ describe('Executions', () => {
     });
 
     describe('in table mode', () => {
+        describe('change visible columns', () => {
+            before(() =>
```
Comment (Gelio) [inline test/cypress/integration/widgets/executions_spec.ts:22]: Minor:

Not sure if we need this additional structure of wrapping the test in `describe` and then adding a `before`. I would just have a single test here `it('allows showing the deployment display name')` and ditch the `before` and the `describe` :slightly_smiling_face:

**Review by Gelio — APPROVED** [review]
Looks good :+1: I did not run it in the UI though

**Review by kubama — APPROVED** [review]

**Review by Gelio — APPROVED** [review]
LGTM :slightly_smiling_face:

**Review by Gelio — APPROVED** [review]
@stepek just FYI, if you're clicking the "Update branch" PR after you get approvals, get approvals are not dismissed:
image

The approvals are only dismissed if you do actual code changes and push them manually. If you just merge with `master` / `6.1.0-build` via GitHub's UI, there's no need to ask for approval :slightly_smiling_face:

**Comment by stepek** [comment]
> @stepek just FYI, if you're clicking the "Update branch" PR after you get approvals, get approvals are not dismissed:
> image
> 
> The approvals are only dismissed if you do actual code changes and push them manually. If you just merge with `master` / `6.1.0-build` via GitHub's UI, there's no need to ask for approval 🙂

Sorry for that, after clicking `update branch` approvals gone for some part of time :)

**Comment by Gelio** [comment]
> Sorry for that, after clicking update branch approvals gone for some part of time :)

No problem :smile: I just wanted to let you know for the future

btw. make sure to clean up after this PR and remove the branch if you don't plan on using it :slightly_smiling_face: 

Also, this PR should be cherry-picked to `6.1.0-build`

**Pull request merged**